recipes/msnoise/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "msnoise" %}
+{% set version = "1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5795f0d452e9ad7728e85d8474b347f40fbc2238273a8145818979791e0725c2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - msnoise = msnoise.scripts.msnoise:run
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
@ThomasLecocq  This should include all dependencies, like they are listed in your setup.py

Otherwise people can do `conda install msnoise` and end up with an installation that doesnt even run cuz of missing deps

```
      install_requires=[
          'setuptools',
          'numpy>=1.0',
          'scipy',
          'pandas',
          'matplotlib',
          'sqlalchemy',
          'obspy',
          'click',
          'pymysql',
          'flask',
          'flask-admin',
          'flask-wtf',
          'markdown',
          'folium',
          'wtforms',
          'jinja2',
          'scandir',  # useful for python < 3.5
          'logbook'
      ],
```

To make a new build on conda-forge:
 - fork https://github.com/conda-forge/msnoise-feedstock to either your account or @ROBelgium
 - make a branch with the changes in recipe meta.yaml (make sure to increment build number)
 - make PR against conda-forge/msnoise-feedstock
 - check CI, merge on green

make sure to not directly push to branches in conda-forge/msnoise-feedstock, any direct pushes will be built and immediately deployed as packages to anaconda.org/conda-forge!

Reply:
ok, doing in https://github.com/conda-forge/msnoise-feedstock/pull/1, switching to there for the next steps. Thanks @megies